app/main/routes_frontend.py

```diff
@@ -17,7 +17,8 @@
 from .recipe_parser import PicoBrewRecipe, PicoBrewRecipeImport, ZymaticRecipe, ZymaticRecipeImport, ZSeriesRecipe
 from .session_parser import load_ferm_session, get_ferm_graph_data, get_brew_graph_data, load_brew_session, active_brew_sessions, active_ferm_sessions
 from .config import base_path, zymatic_recipe_path, zseries_recipe_path, pico_recipe_path, ferm_archive_sessions_path, brew_archive_sessions_path, MachineType
-
+from .recipe_import import import_recipes
+from .config import MachineType

```

Comment:
MachineType is already imported 2 lines above... suggest running pep8/flake8 on all py files in the commit (w/ max-line-length set to something ridiculous or ignored)

Reply:
Results of that pep8 and flake are in the GitHub actions, right now we only fail on a few conditions (no import, and a few others) could look at adding a few more codes to the fail logic

Reply:
I use VSCode (without any linters) and then just run pep8/flake8